utils/utils.go

```diff
@@ -122,6 +125,54 @@ func FilePath(name, ext string, escape bool) (string, error) {
 	return outputPath, nil
 }
 
+// FileLineCounter Counts line in file
+func FileLineCounter(r io.Reader) (int, error) {
+	buf := make([]byte, 32*1024)
+	count := 0
+	lineSep := []byte{'\n'}
+
+	for {
+		c, err := r.Read(buf)
+		count += bytes.Count(buf[:c], lineSep)
+
+		switch {
+		case err == io.EOF:
+			return count, nil
+
+		case err != nil:
+			return count, err
+		}
+	}
+}
+
+// ParseInputFile Parses input file into args
+func ParseInputFile(r io.Reader) ([]string, error) {
```

Comment:
this function will not return an error anymore, so we can change the definition to `ParseInputFile(r io.Reader) []string`